fsh/ig-data/input/pagecontent/index.md

```diff
@@ -6,9 +6,9 @@
 *[API]: Application Programming Interface
 *[CDS]: Clinical Decision Support
 *[CPOE]: Computerized Physician Order Entry
-*[CRUD]: Copy Read Update Delete
+*[CRUD]: Create Read Update Delete
```

Comment:
Wow!  Um, thanks for catching that.  Yikes

Reply:
Ha ha, don't feel bad though - I'm sure all of us have made such mistakes!